LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Qbunjo
```

Comment:
Well, I don't know if another change for DFPlayer only because of lack of license is necessary. I would rather not to do it.
Ad.2 - We use only the interface, and we provide the sources, so everything is ok here.